API/src/microservices/Assessment-Service/index.js

```diff
@@ -0,0 +1,94 @@
+const Assessment = require("../Database/assessment");
+const bookshelf = require("../Database/bookshelf");
+
+//TODO: update created_at, convert Date to prostgreSQL date object
+
+//assessment object
+let assessment = {};
```

Comment:
A general suggestion here is to not use a globally defined object like this.  On an application with multiple users, this would be overwritten instantly and simply wouldn't work.  Because of this, you should pass your assessment object into each function manually.

Reply:
Thanks! Will go ahead and change it. Now that i think about it's completely unnecessary for it to be global.